### PR TITLE
Add Quefrency Rev. 4

### PR DESCRIPTION
**Pull request by nooges**
## Description

Add Quefrency Rev. 4 PCB

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/quefrency/rev4/readme.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Quefrency Rev. 4
+
+Quefrency Rev. 4 features Kailh hotswap sockets with a fixed layout. The left macro portion and right column can be broken off, and split backspace or 2u backspace can be used.
+
+Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)  
+Hardware Availability: [Keebio](https://keeb.io/)
```
Comment (fauxpark) [inline keyboards/keebio/quefrency/rev4/readme.md:6]:
```suggestion
* Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)
* Hardware Availability: [Keebio](https://keeb.io/)
```

File: keyboards/keebio/quefrency/rev4/readme.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    make keebio/quefrency/rev4:default
+
+Example of flashing this keyboard:
+
+    make keebio/quefrency/rev4:default:flash
+
+See [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
```
Comment (fauxpark) [inline keyboards/keebio/quefrency/rev4/readme.md:16]:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/keebio/quefrency/rev4/rev4.c [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    rgblight_enable(); // Enable RGB by default
+    rgblight_sethsv(0, 255, 255);  // Set default HSV - red hue, full saturation, full brightness
+#ifdef RGBLIGHT_ANIMATIONS
+    rgblight_mode(RGBLIGHT_MODE_RAINBOW_SWIRL + 2); // set to RGB_RAINBOW_SWIRL by default
+#endif
+#endif
+
+    eeconfig_update_kb(0);
+    eeconfig_init_user();
+}
```
Comment (fauxpark) [inline keyboards/keebio/quefrency/rev4/rev4.c:30]: Should be doable with defines as of #11912

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/keebio/quefrency/rev4/rev4.c (hunk 3 above)
Comment (nooges) [reply keyboards/keebio/quefrency/rev4/rev4.c]: Totally forgot about this new option again

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**